src/server/routes/projects.js

```diff
@@ -754,14 +755,26 @@ module.exports = [
         URL: 'projects/:owner',
         middleware: ['setUsername'],
         Handler: function(req, res) {
-            var publicOnly = req.params.owner !== req.session.username;
+            const publicOnly = req.params.owner !== req.session.username;
```

Comment:
two small stylistic suggestions: 
1. `publicOnly` isn't clear to me (doesn't directly reflect the check), I would add a comment 'show only public projects' or have a variable as `isOwner` or something to show that owner is requesting.
edit: after going through it again it makes more sense now but I'm I'm not sure which one would be clearer to unfamiliar eyes
2.  single quotes for ln:766